lib/screens/add_time_screen.dart

```diff
@@ -51,37 +52,49 @@ class _AddTimeScreenState extends State<AddTimeScreen> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-        backgroundColor: Color.fromRGBO(228, 230, 195, 1),
+        backgroundColor: Color.fromRGBO(188, 231, 253, 1),
         appBar: AppBar(
-          backgroundColor: Color.fromRGBO(137, 152, 120, 1),
-          title: const Text("Create"),
+          backgroundColor: Color.fromRGBO(175, 59, 110, 1),
+          title: const Text(
+            "Create",
+            style: TextStyle(color: Color.fromRGBO(186, 103, 144, 1)),
+          ),
         ),
         body: Column(
           mainAxisAlignment: MainAxisAlignment.end,
           children: [
             Container(
               decoration: const BoxDecoration(
                 borderRadius: BorderRadius.all(Radius.circular(10.0)),
-                color: Color.fromRGBO(137, 152, 120, 1),
+                color: Color.fromRGBO(175, 59, 110, 1),
```

Comment:
colors should be moved to a different class as constants
+prefer using HEX color constructor